src/main/java/de/hysky/skyblocker/skyblock/tabhud/util/Colors.java

```diff
@@ -10,18 +9,6 @@ public class Colors {
      * @return an int representing a color, where 100% = green and 0% = red
      */
     public static int pcntToCol(float pcnt) {
-        return MathHelper.hsvToRgb(pcnt / 300f, 0.9f, 0.9f);
-    }
-
-    public static Formatting hypixelProgressColor(float pcnt) {
-        if (pcnt < 25) {
-            return Formatting.RED;
-        } else if (pcnt < 50) {
-            return Formatting.GOLD;
-        } else if (pcnt < 75) {
-            return Formatting.YELLOW;
-        } else {
-            return Formatting.GREEN;
-        }
+        return MathHelper.hsvToRgb(pcnt / 300, 1, 1);
```

Comment:
why change it to 1?

Reply:
To make it the same with vanilla colors at either end. (Red is 255, 0, 0 and green is 0, 255, 0.) Also durability also uses this gradient.